10up/wp-vulnerability-scanner
# Add vulnerability severity data to command output.

### Description of the Change
PR adds severity data of vulnerability in command output.

![image](https://user-images.githubusercontent.com/10613171/229153629-86180a80-3fe9-45bb-9a1a-197c8cff0cdb.png)


Closes #85 

### How to test the Change
1. Configure this scanner repo and configure API provider
2. Run commands like `wp vuln status`, `wp vuln theme-status` etc... and make sure severity details are there.

### Changelog Entry

> Added - Vulnerability severity data to command output.


### Credits

Props @iamdharmesh 


### Checklist:
